pkgs/development/coq-modules/compcert/default.nix

```diff
@@ -26,8 +26,9 @@ let compcert = mkCoqDerivation rec {
     "3.10".sha256 = "sha256:19rmx8r8v46101ij5myfrz60arqjy7q3ra3fb8mxqqi3c8c4l4j6";
   };
 
+  mlPlugin = true;
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = with ocamlPackages; [ ocaml findlib menhir menhirLib ] ++ [ coq coq2html ];
+  buildInputs = with coq.ocamlPackages; [ menhir menhirLib ] ++ [ coq2html ];
```

Comment:
Why these changes? CompCert is not a plugin for Coq.

Moreover, I want to compile CompCert with OCaml 5.0 even if Coq is stuck with OCaml 3.12.

Reply:
I think this is a side effect of my previous mistake, I will fix ASAP. Thanks for your eagle eye.

Reply:
fixed